bears/generate_package.py

```diff
@@ -120,6 +121,17 @@ def main():
             bear_name, _ = os.path.splitext(os.path.basename(bear_file_name))
             create_file_structure_for_packages(
                 os.path.join('bears', 'upload'), bear_file_name, bear_name)
+            if bear_object.REQUIREMENTS:
+                for requirement in bear_object.REQUIREMENTS:
+                    if isinstance(requirement, PipRequirement):
+                        with open(os.path.join(
+                                    'bears', 'upload',
+                                    bear_name, 'requirements.txt'),
+                                  'a') as reqtxt:
```

Comment:
Oh ok. Thought it might save some performance but can't think about a nice way...
You could first check if there are pip requirements then open then iterate again. Depends on cost of iterate vs open but probably doesn't matter :)